src/webui/api/rsscontroller.cpp

```diff
@@ -139,3 +166,26 @@ void RSSController::rulesAction()
 
     setResult(jsonObj);
 }
+
+void RSSController::matchingArticlesAction()
+{
+    requireParams({"ruleName"});
+
+    const QString ruleName {params()["ruleName"].trimmed()};
```

Comment:
Is `trimmed()` really required?

Reply:
Yes, you are right it's not required. I actually copied the line from `RSSController::removeRuleAction()` where it is also not required. Should I create a PR removing unnecessary trimms?